src/plugins/comicsPlayer/plugin.js

```diff
@@ -21,12 +22,28 @@ export class ComicsPlayer {
 
         this.onDialogClosed = this.onDialogClosed.bind(this);
         this.onWindowKeyUp = this.onWindowKeyUp.bind(this);
+
+        const storedPrefs = localStorage.getItem('comicsPlayerPrefs');
+        if (!storedPrefs)
+            localStorage.setItem('comicsPlayerPrefs', '{}');
```

Comment:
I think this should probably use `userSettings` instead of dealing directly with `localStorage`.

https://github.com/jellyfin/jellyfin-web/blob/master/src/scripts/settings/userSettings.js

Reply:
Sure, I will try this out.

Reply:
Updated PR with the requested change, please review this again.